src/main/resources/db/migration/V202409231010__create_table_feedback.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE feedback
+(
+    id SERIAL PRIMARY KEY ,
+    company_id BIGINT NOT NULL,
+    employee_id BIGINT NOT NULL ,
```

Comment:
if it's annoymous, are we going to save the id? I'm not sure. I missed that in the HLD. So maybe we want to allow it to be Null

Reply:
yes it should be null in this case i added migration for this now .

Reply:
but I see: NOT NULL :). maybe you forgot to remove it?

Reply:
I DID ANOTHER MIGRATION FOR THIS TO DROP THE NOT NULL